BaseCollector.py

```diff
@@ -4,9 +4,14 @@
 import os
 from tools.helper import yaml_read
 from tools.Resources import Resources
+from prometheus_client.core import GaugeMetricFamily, InfoMetricFamily, UnknownMetricFamily
+
 
 class BaseCollector(ABC):
 
+    def __init__(self):
+        self.vrops_entity_name = 'base'
```

Comment:
this does not seem to be used, any specific reason for this?

Reply:
This is used in `describe` method and will be overwritten in every collector.